pkgs/top-level/all-packages.nix

```diff
@@ -23705,6 +23705,10 @@ with pkgs;
 
   cipher = callPackage ../applications/misc/cipher { };
 
+  ciscoPacketTracer7 = callPackage ../applications/networking/cisco-packet-tracer/cisco-packet-tracer7.nix { };
```

Comment:
Name files as version only (ie. `7.nix`) to avoid repetition, `cisco-packet-tracer` is already in folder.

Reply:
Done